sftp-config.json

```diff
@@ -0,0 +1,45 @@
+{
+    // The tab key will cycle through the settings when first created
+    // Visit https://codexns.io/products/sftp_for_subime/settings for help
+    
+    // sftp, ftp or ftps
+    "type": "sftp",
+
+    "save_before_upload": true,
+    "upload_on_save": false,
+    "sync_down_on_open": false,
+    "sync_skip_deletes": false,
+    "sync_same_age": true,
+    "confirm_downloads": false,
+    "confirm_sync": true,
+    "confirm_overwrite_newer": false,
+    
+    "host": "public.websitetoolbox.com",
+    "user": "kishanm",
+    "password": "dh2oZf7SSk",
```

Comment:
Are you kidding me? You included this file in the pull request and even @masihur did not notice?

Do you understand how big of a security issue you would have created? Completely irresponsible. 

Properly review this pull request. 

@masihur - ?????

Reply:
Really sorry about silly mistake. Actually this file is in my gitignore after my commit it first time. Then I forgot to remove it from repository. I have removed this file.